airplonk

```diff
@@ -35,7 +35,9 @@ declare -i ERR_COUNT=0  # Number of times 0 nodes where discovered.
 declare -i SPINNER_PID  # PID of the 'looking for nodes...' spinner process.
 declare -i COLUMNS      # Number of columns of the terminal window. (declare
 COLUMNS="$(tput cols)"  # and assign seperately to avoid masking return values)
-declare os="$OSTYPE"    # Determine environment.
+declare OS="$(uname -s)"    # Determine environment.
```

Comment:
Thanks for the review comment, so the feeling here was it was more responsive than $OSTYPE based on some initial testing in the terminal. "$ $OSTYPE" seems to return "linux-gnu: Command not found " when used directly in the terminal so my worry there was this could be interpreted as an error without $OSTYPE being quietened against errors? I then found "uname" and its flags which seemed to give more granular control as to what you want to know about the OS across Linux, Mac, BSD, etc.